### ISSUE TITLE
svgs break the runner

### ISSUE THREAD

**Issue by nothingrandom** [liamfiddler/eleventy-plugin-lazyimages#9]
If you have svgs in your repo and are using them with an `img` (not the best option, but I've done this before. Jimp fails to run over svg images and returns `Error: Could not find MIME for Buffer <null>`, due to the fact it supports `jpeg`, `png`, `tiff`, `gif` and `bmp`.

**Comment by nothingrandom** [comment]
I had one thought pattern to fix this currently, that being in the `processImage` function to do a check if svg continue, otherwise do nothing.

```js
const processImage = async imgElem => {
  if (imgPath.substr(imgPath.length - 3) !== 'svg') {
    // setAtrributes
    // try await getImageData
    // catch
  }
};
```

**Comment by liamfiddler** [comment]
Good find! Looks like it's not just .svg files, but also .webp and any other format that isn't supported by JIMP.

I've created a PR inspired by your suggested fix - https://github.com/liamfiddler/eleventy-plugin-lazyimages/pull/10/files

Before I merge it I'd love to get your feedback; do you think the plugin should:
1. Ignore unsupported image types - it will not set any attributes for unsupported images.
2. Set all the attributes for the image except `width`, `height`, & `src` (the ones that depend on JIMP). This means images can still be handled by the lazysizes script, they just don't get the low-res placeholder image.

**Comment by nothingrandom** [comment]
I'd have the expectation for 2! Images should still get the lazysizes, just without the placeholder.

Secondly, do you know of https://github.com/lovell/sharp? From a really quick look it certainly handles `webp` and they've tagged it with svg, so might handle those too?

**Comment by liamfiddler** [comment]
OK, I've updated the PR to work that way. It'll likely be merged into master tomorrow morning 👍 

When I first created the plugin it was using Sharp, but I had problems with Sharp not installing correctly on certain machines. While debugging the installation issues I discovered JIMP. Although JIMP supports fewer file types, it handles the most common ones without requiring any external or native dependencies. JIMP became the quickest and easiest solution to getting the plugin released.

Now that it's released I'm open to exploring other options, and support for SVG + Webp are pretty compelling reasons to try Sharp again. I'll make a branch and test it out. Thanks!

**Comment by nothingrandom** [comment]
Fantastic, thanks for the quick PR work!

**Comment by karlnippoldt** [comment]
I’m encountering this issue, too, even with the fix in github. I think NPM isn’t giving me the latest version since there hasn’t been a version number bump.

**Comment by liamfiddler** [comment]
This fix for this has been [released in v1.1.0](https://www.npmjs.com/package/eleventy-plugin-lazyimages/v/1.1.0)  🎉
